i2c/ssd1306_i2c/ssd1306_font.h

```diff
@@ -7,42 +7,44 @@
 // Vertical bitmaps, A-Z, 0-9. Each is 8 pixels high and wide
 // Theses are defined vertically to make them quick to copy to FB
 
+// Update: Reverse font
```

Comment:
This comment probably doesn't make sense when viewed outside the context of this PR? (i.e. it should probably be deleted?)

Reply:
Yes, it should be deleted in the context of this PR.